src/main/kotlin/io/github/inductiveautomation/kindling/localization/TranslationTool.kt

```diff
@@ -0,0 +1,433 @@
+package io.github.inductiveautomation.kindling.localization
+
+import com.formdev.flatlaf.extras.FlatSVGIcon
+import io.github.inductiveautomation.kindling.core.ClipboardTool
+import io.github.inductiveautomation.kindling.core.Kindling.Preferences.General.HomeLocation
+import io.github.inductiveautomation.kindling.core.Kindling.Preferences.UI.Theme
+import io.github.inductiveautomation.kindling.core.MultiTool
+import io.github.inductiveautomation.kindling.core.ToolPanel
+import io.github.inductiveautomation.kindling.localization.TranslationTool.exportZipFileChooser
+import io.github.inductiveautomation.kindling.utils.Action
+import io.github.inductiveautomation.kindling.utils.Column
+import io.github.inductiveautomation.kindling.utils.ColumnList
+import io.github.inductiveautomation.kindling.utils.FileFilter
+import io.github.inductiveautomation.kindling.utils.FlatScrollPane
+import io.github.inductiveautomation.kindling.utils.ReifiedJXTable
+import io.github.inductiveautomation.kindling.utils.ReifiedTableModel
+import io.github.inductiveautomation.kindling.utils.clipboardString
+import io.github.inductiveautomation.kindling.utils.configureCellRenderer
+import net.miginfocom.swing.MigLayout
+import java.awt.BorderLayout
+import java.awt.Toolkit
+import java.awt.event.KeyEvent
+import java.io.InputStream
+import java.nio.file.Path
+import java.time.LocalTime
+import java.time.format.DateTimeFormatter
+import java.util.Locale
+import java.util.Properties
+import java.util.SortedMap
+import java.util.TreeMap
+import java.util.zip.ZipEntry
+import java.util.zip.ZipOutputStream
+import javax.swing.Icon
+import javax.swing.JButton
+import javax.swing.JComboBox
+import javax.swing.JFileChooser
+import javax.swing.JOptionPane
+import javax.swing.JPanel
+import javax.swing.KeyStroke
+import javax.swing.ListSelectionModel
+import javax.swing.filechooser.FileNameExtensionFilter
+import javax.swing.table.AbstractTableModel
+import kotlin.io.path.absolutePathString
+import kotlin.io.path.extension
+import kotlin.io.path.inputStream
+import kotlin.io.path.name
+import kotlin.io.path.nameWithoutExtension
+import kotlin.io.path.outputStream
+
+private data class TableRow(
+    val term: String,
+    val translationMap: MutableMap<Locale, String?>,
+)
+
+class TranslationView(
+    private val bundles: SortedMap<Locale, Properties>,
```

Comment:
"Constructor Parameter is never used as a Property" warning?

Reply:
Ooh, good catch, yeah, that was leftover from an older version before I made the custom model class.